cmd/sync/config.go

```diff
@@ -11,14 +11,13 @@ import (
 type commonConfig struct {
 	APIEndpoint           string        `yaml:"api_endpoint"`
 	SyncIntervalInSeconds time.Duration `yaml:"sync_interval_in_seconds"`
+	CloudProvider         string        `yaml:"cloud_provider"`
 }
 
 const errorMsgFormat = "The mandatory field %v is either empty or missing in the config file"
 const intervalErrorMsg = "The mandatory field sync_interval_in_seconds is either 0 or missing in the config file"
-const upstreamNameErrorMsg = "The mandatory field name is either empty or missing for an upstream in the config file"
-const upstreamErrorMsgFormat = "The mandatory field %v is either empty or missing for the upstream %v in the config file"
-const upstreamPortErrorMsgFormat = "The mandatory field port is either zero or missing for the upstream %v in the config file"
-const upstreamKindErrorMsgFormat = "The mandatory field kind is either not equal to http or tcp or missing for the upstream %v in the config file"
+const cloudProviderErrorMsg = "Invalid Cloud Provider: %v"
```

Comment:
For consistency with the other messages
```suggestion
const cloudProviderErrorMsg = "The field cloud_provider has invalid value %v in the config file"
```